tfjs-backend-webgpu/package.json

```diff
@@ -15,13 +15,9 @@
     "bundle": "rollup -c",
     "bundle-ci": "rollup -c --ci",
     "build-link-package": "cd ../link-package && yarn build",
```

Comment:
`yarn build` will also build `yarn build-backend-webgl && yarn build-tflite` in link-package. Maybe we need a new one, which only includes `yarn build-backend-cpu && yarn build-core`.
@lina128 @jinjingforever What's your opinion?